src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -118,25 +125,17 @@ private static Task RunCommandAsync(UpgradeOptions options, Action<HostBuilderCo
 
                     services.AddSingleton(new InputOutputStreams(Console.In, Console.Out));
                     services.AddSingleton<CommandProvider>();
-                    services.AddSingleton(logSettings);
+                    services.TryAddSingleton(new LogSettings(true));
```

Comment:
Should this go in the integration test's UpgradeRunner so that all necessary services are registered there?

Reply:
There's a command that requires it so this will add a default one if needed